docs/source/getting-started/tabular-data/computing-basic-statistics.ipynb

```diff
@@ -0,0 +1,472 @@
+      "  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0\n",
+      "100 5933k  100 5933k    0     0  1022k      0  0:00:05  0:00:05 --:--:-- 1100k\n",
+      "Archive:  data.zip\n",
+      "  inflating: sample_FR_LFS.csv       \n",
+      "  inflating: __MACOSX/._sample_FR_LFS.csv  \n"
+     ]
+    }
+   ],
+   "source": [
+    "![ -e sample_FR_LFS.csv ] || ( curl 'https://github.com/opendp/dp-test-datasets/blob/master/data/sample_FR_LFS.csv.zip?raw=true' --location --output sample_FR_LFS.csv.zip; unzip sample_FR_LFS.csv.zip )\n",
+    "\n",
+    "df = pl.scan_csv(\"sample_FR_LFS.csv\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The compositor here is nearly identical to the one explained in the introduction. One additional parameter that is included is the `max_num_partitions.` This is required when the metric isn't sensitive to ordering. In the following examples, there is only one partition in select, so `max_num_partitions` is set to 1. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<!-- TODO\\n\",\n",
+    "    \"For further information on the context parameters, see the introduction for justification. \\n\",\n",
+    "    \"-->"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Filter HWUSUAL of null values. \n",
+    "df = df.filter(pl.col(\"HWUSUAL\") != 99)\n",
+    "\n",
+    "estimated_max_partition_len = 60_000_000\n",
+    "\n",
+    "context = dp.Context.compositor(\n",
+    "    data=df,\n",
+    "    privacy_unit=dp.unit_of(contributions=36),\n",
+    "    privacy_loss=dp.loss_of(epsilon=1.0),\n",
+    "    split_evenly_over=5,\n",
+    "    margins={\n",
+    "        (): dp.Margin(public_info= \"lengths\", max_partition_length=estimated_max_partition_len, max_num_partitions=1),\n",
+    "    },\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In these examples we see some differences between how regular Polars and Polars with DP are used:\n",
+    "\n",
+    "### 1. Specifying the Data\n",
+    "\n",
+    "In a regular query, use `df` directly. In a differentially private query, use `context.query()`. \n",
+    "\n",
+    "### 2. Applying the Method\n",
+    "\n",
+    "In a regular query, you'll often be able to apply the function directly, such as `.sum()`. In a differentially private query, you may need to pass additional parameters. For example, to compute the differentially private sum, pass in the bounds as a tuple: `dp.sum((1,10))`. \n",
+    "\n",
+    "### 3. Collecting the Results \n",
+    "\n",
+    "In a regular query, use `collect()`. In a differentially private query, use `release().collect().`. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Sum"
```

Comment:
Make it clear that the first value for the number of hours being output is a non-private value even before you write the code snippet to show the reader clearly what is going to happen next.